src/styles/blocks/product_card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  width: $card-width;
+  padding: 32px 16px 12px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+
+    display: block;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+
+    font-weight: $font-w-s;
+    font-size: 12px;
+    line-height: 18px;
+    color: $card-text-color;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+
+    font-weight: $font-w-n;
```

Comment:
```suggestion
   
    ```
 this rule **font-weight: 400;** work by default
 for all font-families. you don't need it.
Remove it from **_variables.scss** too